app/uk/gov/hmrc/helptosave/actors/UCThresholdManager.scala

```diff
@@ -17,26 +17,22 @@
 package uk.gov.hmrc.helptosave.actors
 
 import configs.syntax._
-import akka.actor.{Actor, ActorRef, Props, Scheduler, Stash}
+import akka.actor.{Actor, ActorRef, Props, Scheduler}
 import uk.gov.hmrc.helptosave.actors.UCThresholdManager._
 import akka.pattern.pipe
 import akka.pattern.ask
 import akka.util.Timeout
 import com.typesafe.config.Config
-import uk.gov.hmrc.helptosave.util.{Logging, PagerDutyAlerting, WithExponentialBackoffRetry}
+import uk.gov.hmrc.helptosave.util.{Logging, PagerDutyAlerting, Time, WithExponentialBackoffRetry}
 
 import scala.concurrent.duration._
 import scala.concurrent.Future
-
 import UCThresholdConnectorProxy.{GetThresholdValue ⇒ GetDESThresholdValue, GetThresholdValueResponse ⇒ GetDESThresholdValueResponse}
```

Comment:
do we still need these shady names in this import ?

Reply:
Yeah we need to keep this to differentiate between case classes in the UCThresholdManager object and case classes in the UCThresholdConnectorProxy object